server/views/probationPractitionerReferrals/interventionProgress.njk

```diff
@@ -36,7 +36,63 @@
   </p>
 
   {% if presenter.hasSessions %}
-    {{ govukTable(sessionTableArgs(govukTag)) }}
```

Comment:
this has random indents, can you make sure it's formatted so blocks line up correctly, we don't have a formatter for nunjucks so this needs to be manually done